app/src/main/java/net/gsantner/markor/widget/TodoWidgetRemoteViewsFactory.java

```diff
@@ -86,4 +94,84 @@ public long getItemId(int position) {
     public boolean hasStableIds() {
         return false;
     }
+
+    private Spannable getTaskSpannable(TodoTxtTask task) {
```

Comment:
This static function will return a highlighter callable which can apply todotxt highlighting to a spannable.

https://github.com/gsantner/markor/blob/master/app/src/main/java/net/gsantner/markor/frontend/MarkorDialogFactory.java#L758

So simply 
```
final GsCallback.a1<Spannable> hl = getSttHighlighter();
for (Spannable s : lines) {
    hl.callback(s)
}
```

Reply:
This will highlight every line in `lines` 

You will need to make getSttHighlighter public and move it somewhere appropriate.

Reply:
```
final GsCallback.a1<Spannable> hl = getSttHighlighter();
Spannable s = new SpannableString(_tasks.get(position).getLine());
hl.callback(s);
views.setTextViewText(R.id.todo_widget_item_text, s);
```
I couldn't make it work. What is wrong here?

It's also very important to remove creation dates, priority parentheses, etc. as they take up valuable space especially on narrower screens. How would that be achieved?,